markhor_slam/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(markhor_slam)
+
+find_package(catkin REQUIRED)
+
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES markhor_slam
+#  CATKIN_DEPENDS other_catkin_pkg
+#  DEPENDS system_lib
+)
+
+include_directories(
+# include
```

Comment:
Same as the previous comment, you can remove the comment here also. You might want to try to remove the whole function call, since we are not really using any directories for our package. I think this is different from `catkin_package`, because it does some operation in the background for the ROS aspect of the build.